### ISSUE TITLE
10-7959c: Remove extra signer screens

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#83140]

**Comment by michaelclement** [comment]
@marywang2 @jamiefiore Just want to make sure I'm understanding correctly - we're going to drop the "your information" question, and just assume that the person filling out 10-7959c is the applicant?

Main reason I'm asking is because it has two repercussions later in the form
1. The headings will now always say "**Your** health insurance" etc, rather than ever saying the applicant's full name since the user is assumed to be the applicant
2. The signature block on the review page will now always expect the applicant's full name (not the third party signer/veteran name, as we won't be collecting that info)


Is my understanding correct?

**Comment by jamiefiore** [comment]
@michaelclement I discussed the "your" language with the design team and we didn't think it was worth adding the extra question in to deal with that situation so we are ok with leaving it. The headers on the rest of the form should be populate the applicant's first name so it should say: Jamie's mailing address, Jamie's health insurance.

The bigger UX issue in the signature. Is this a way to remove that restriction of having to sign with the applicant'
s name?

**Comment by marywang2** [comment]
@jamiefiore let's see if we can explore some other options first.

**Comment by michaelclement** [comment]
@jamiefiore Thanks for clarifying that we're dropping "your" - I think I knew that and just forgot. As for removing the signature block entirely, I don't know if we can do that (from a rules standpoint). I wonder if we're allowed to just have an arbitrary name entered in that field though? CC @marywang2

**Comment by michaelclement** [comment]
@marywang2 I've gone ahead and implemented this [in this PR here](https://github.com/department-of-veterans-affairs/vets-website/pull/29969). After we've done some more research/gotten a design for a new certifier info page, I figure we should probably just open a separate ticket to add whatever new interactions we need in. Does that seem reasonable? CC @jamiefiore

**Comment by michaelclement** [comment]
Waiting on 
- [X] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
@marywang2 Merged, should be on staging shortly and ready for review. Thanks!
